logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -277,4 +316,52 @@ class MLSConversationDataSource(
         conversationDAO.getAllMembers(conversationID).first().map { idMapper.fromDaoModel(it.user) }
     }
 
+    private suspend fun retryOnCommitFailure(groupID: GroupID, operation: suspend () -> Either<CoreFailure, Unit>) =
```

Comment:
since it's not always a retry, can we rename it to something like wrapStorageRequest, e.g. wrapMLSCommitRequest?

Reply:
It's always a retry if does something otherwise it passes the error on, maybe it would be more accurate to name it `retryOnCommitFailureIfPossible` but I find it too wordy. 

IMO this is not the same as `wrapStorageRequest ` we aren't catching exceptions putting them in a `Either<CoreFailure, Unit>`.